jira_view.py

```diff
@@ -25,45 +26,46 @@ def get_ctx_result(provides, result):
     summary = result.get_summary()
     data = result.get_data()
 
-    ctx_result['param'] = param
+    ctx_result["param"] = param
 
     if summary:
-        ctx_result['summary'] = summary
+        ctx_result["summary"] = summary
 
-    ctx_result['action'] = provides
+    ctx_result["action"] = provides
 
     if not data:
```

Comment:
We can simplify 36 - 39 as 
`ctx_result["data"] = data if data else []`

Reply:
Good catch, applied